### PR TITLE
Java SDK issue 1100 - Zuul and Spring landing page

### PR DESCRIPTION
**Pull request by mrioan**
IMPORTANT NOTE: this must not be released until Java SDK 1.3.0 is out

### CODE REVIEW

File: source/_layouts/javapage.html [hunk 1]
```diff
@@ -70,6 +72,80 @@ <h2 class="heading-blc">Java</h2>
...
+                </div>
+              </div>
+              <div class="box-container">
+                  <div class="top-section-grp">
+                    <div class="heading-block">
+                      <h2>Spring WebMVC</h2>
+                    </div>
+                    <div class="two-boxes-blc">
+                      <div class="row">
+                        <a href="/java/spring/quickstart.html">
```
Comment (dogeared) [inline source/_layouts/javapage.html]: please update to `/java/spring-web/quickstart.html`. This keeps things consistent as the spring boot side of things is `spring-boot-web`

File: source/_layouts/javapage.html [hunk 2]
```diff
@@ -70,6 +72,80 @@ <h2 class="heading-blc">Java</h2>
...
+                          <div class="col-md-6 col-xs-12">
+                            <div class="single-bx">
+                              <div class="icon"><i class="fa fa-forward" aria-hidden="true"></i></div>
+                              <div class="text">Quickstart</div>
+                            </div>
+                          </div>
+                        </a>
+                        <a href="/java/spring/">
```
Comment (dogeared) [inline source/_layouts/javapage.html]: please update to `/java/spring-web/`. this keeps is consistent with spring boot where we have `spring-boot-web`

File: source/_layouts/javapage.html [hunk 3]
```diff
@@ -70,6 +72,80 @@ <h2 class="heading-blc">Java</h2>
...
+                              <div class="text">Documentation</div>
+                            </div>
+                          </div>
+                        </a>
+                      </div>
+                    </div>
+                    <ul class="list-item">
+                      <li><a href="https://github.com/stormpath/stormpath-sdk-java/tree/master/examples/spring-security-webmvc"> <div class="icon"><i class="fa fa-angle-left" aria-hidden="true"></i> <i class="fa fa-angle-right" aria-hidden="true"></i></div>Example Application</a></li>
+                      <li><a href="https://github.com/stormpath/stormpath-sdk-java/tree/master/extensions/spring"><div class="icon"> <i class="fa fa-globe" aria-hidden="true"></i> </div>Project on GitHub</a></li>
+                      <li><a href="/java/spring/tutorial.html"> <div class="icon"> <i class="fa fa-pencil" aria-hidden="true"></i> </div>Webapp Tutorial</a></li>
```
Comment (dogeared) [inline source/_layouts/javapage.html]: please update to `/java/spring-web/tutorial.html`. this keeps it consistent with spring boot where we use `spring-boot-web`

**Comment by mrioan** [comment]
@dogeared I did the changes you requested. Is this PR approved?

**Pull request merged**